src/components/gltf-model-plus.js

```diff
@@ -31,57 +31,54 @@ AFRAME.GLTFModelPlus = {
   }
 };
 
-// From https://gist.github.com/cdata/f2d7a6ccdec071839bc1954c32595e87
-// Tracking glTF cloning here: https://github.com/mrdoob/three.js/issues/11573
-function cloneGltf(gltf) {
-  const skinnedMeshes = {};
-  gltf.scene.traverse(node => {
-    if (!node.name) {
-      node.name = node.uuid;
-    }
-    if (node.isSkinnedMesh) {
-      skinnedMeshes[node.name] = node;
-    }
-  });
+function parallelTraverse(a, b, callback) {
```

Comment:
it took me a bit to figure out what was going on here. i'd probably push this into a domain-specific function that fills the map recursively unless this routine is going to be used elsewhere. alternatively, if you want to keep it generic, i'd write some comments.

is there a reason you couldn't just look this up by id in the cloned graph? just perf?